### PR TITLE
Add directional idle sprites & update camera limits

### PR DESCRIPTION
**Pull request by khatakslied**

### CODE REVIEW

File: assets/room_type/crew_quarters.tres [hunk 1]
```diff
@@ -9,6 +9,6 @@ name = "Crew Quarters"
 price = 100
 powerConsumption = 0
 capacity = 0.0
-minTiles = 4
```
Comment (khatakslied) [inline assets/room_type/crew_quarters.tres:12]: I don't know why these keep getting reset...

File: scripts/Camera2D.gd [hunk 2]
```diff
@@ -63,5 +63,6 @@ func _process(delta):
 	
 	# Clamp the camera's position to the desired boundaries
 	# TODO: Update boundaries
-	position.x = clamp(position.x, 0, 100)
```
Comment (khatakslied) [inline scripts/Camera2D.gd:66]: The position and zoom don't work well together. The default zoom is 0.25x with limits of (0.2-1.0x). Currently, the camera zooms to the left, possibly due to the "Anchor Mode", but when I changed the setting the point of selection doesn't match the cursor position. I'll look into this again later.

File: assets/sprites/idle_agatha.png
Comment (khatakslied) [inline assets/sprites/idle_agatha.png:1]: She's Agatha until we come up with something more suitable.

**Review by conalli — APPROVED** [review]
LGTM

File: assets/sprites/idle_agatha.png
Comment (conalli) [reply assets/sprites/idle_agatha.png:1]: I had been thinking samantha, but agatha works

**Pull request merged**